ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -97,4 +99,47 @@ trait CommonFtpSourceSpec extends BaseSpec {
     }
   }
 
+  "FtpIOSink" should {
```

Comment:
is it worth having a spec showing the "sad path?"  maybe an IOException when trying to write to a location without sufficient permissions?